packages/bundler/web/index.html

```diff
@@ -5,6 +5,10 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<link rel="preconnect" href="https://fonts.gstatic.com" />
 		<link rel="icon" type="image/png" href="/favicon.png" />
+		<link
+			href="https://unpkg.com/react-reflex@4.0.0/styles.css"
+			rel="stylesheet"
+		/>
```

Comment:
Let's not load it from the internet, it might break Remotion working offline and makes the preview unnecessarily slower. We can probably import the CSS in the Preview using an import statement.

Reply:
When I try to `import 'react-reflex/styles.css';`, I get this error: `Error: Cannot find module 'react-reflex/styles.css'`

![image](https://user-images.githubusercontent.com/13774309/111957039-cd39d880-8aeb-11eb-9c21-df267d3730b3.png)

We need to tweak the Webpack config somewhere. I'll try to have a look later but if you already know the answer, please let me know.

Reply:
I see, tried it out and definitely tricky.. ended up inlining and adjusting Webpack config.